amethyst_window/src/system.rs

```diff
@@ -60,20 +60,21 @@ impl WindowSystem {
 }
 
 impl<'a> System<'a> for WindowSystem {
-    type SystemData = WriteExpect<'a, ScreenDimensions>;
+    type SystemData = (WriteExpect<'a, ScreenDimensions>, ReadExpect<'a, Window>);
 
-    fn run(&mut self, mut screen_dimesnions: Self::SystemData) {
-        self.manage_dimensions(&mut screen_dimesnions);
+    fn run(&mut self, (mut screen_dimesnions, window): Self::SystemData) {
```

Comment:
i'm amused how you managed to keep the `dimesnions` spelling 😛

Reply:
This is what happens when the editor allows you to carry such edits quickly without even reading the code 😂